trin-storage/src/sql.rs

```diff
@@ -13,7 +13,8 @@ pub const CREATE_QUERY_DB: &str = "CREATE TABLE IF NOT EXISTS content_data (
                             DROP INDEX IF EXISTS content_size_idx;
                             CREATE INDEX IF NOT EXISTS content_size_idx_2 ON content_data(network, content_size);
                             DROP INDEX IF EXISTS content_id_short_idx;
-                            CREATE INDEX IF NOT EXISTS content_id_short_idx_2 ON content_data(network, content_id_short, content_id_long);
+                            DROP INDEX IF EXISTS content_id_short_idx_2;
+                            CREATE INDEX IF NOT EXISTS content_id_short_idx_3 ON content_data(network, content_id_short);
```

Comment:
Can you explain why this change improved performance?

Reply:
![image](https://github.com/ethereum/trin/assets/31669092/9f8ecd62-0fc8-4eca-ae80-0ce72b3aa765)
It improved performance because the index is indexing the 2 fields this query has to manipulate to get the result.

containing all 3 decreases performance because content_id_long isn't need during the commutation it is just returned at the end. So it is just taking up time as the query goes through.

Lets say you have a query X which is selecting A ordering B and only performing the option on C. In this query we aren't performing any commutations on A. But because it is there we will still check at every step this check takes time. this will also make inserts longer cause the index takes more work has more variables to commute